ff/src/biginteger/macros.rs

```diff
@@ -12,40 +12,71 @@ macro_rules! bigint_impl {
         impl BigInteger for $name {
             const NUM_LIMBS: usize = $num_limbs;
 
-            #[inline]
+            #[ark_ff_asm::unroll_for_loops]
```

Comment:
Can there be a before/after benchmark for field arithmetic on a 256, 384 and 768 bit fields?

Reply:
Adding one now.